packages/avatar-group/src/vaadin-avatar-group-mixin.js

```diff
@@ -476,10 +480,20 @@ export const AvatarGroupMixin = (superClass) =>
       this.__oldOpened = opened;
     }
 
+    /** @private */
+    _updateOverlay() {
+      this.$.overlay.requestContentUpdate();
+    }
+
     /** @private */
     __overflowItemsChanged(items, oldItems) {
+      // Prevent renderer from being called unnecessarily on initialization
+      if (items && items.length === 0 && (!oldItems || oldItems.length === 0)) {
+        return;
+      }
+
       if (items || oldItems) {
-        this.$.overlay.requestContentUpdate();
+        this._updateOverlay();
```

Comment:
I wonder if it's worth extracting this method... could the tests spy on requestContentUpdate instead?

Reply:
Changed to spy on the `__overlayRenderer` which is the actual renderer function.